src/auro-flight-main.js

```diff
@@ -32,6 +34,8 @@ class AuroFlightMain extends LitElement {
       arrivalStation:   { type: String },
       departureTime:    { type: String },
       departureStation: { type: String },
+      oldDepartureStation: {type: String},
```

Comment:
```suggestion
      reroutedDepartureStation: {type: String},
```

Playing with this, using `old` really felt lazy to me. What do you feel about updating to `rerouted`?

Looking at the screenshot attached to the issue, what about rerouted times? Don't those need to change too?

Reply:
Rerouted times are addressed in the departureHeader and arrivalHeader slots. I will rename the attribute rerouted.